infermedica_api/webservice.py

```diff
@@ -479,6 +479,64 @@ def risk_factors_list(self):
 
         return models.RiskFactorList.from_json(response)
 
+    def red_flags(self, diagnosis_request, interview_id=None):
```

Comment:
This function need also to accept `max_results` parameter with default value of 8 (the same way suggest method does).

Reply:
Sure, I can add it. However, it is not mentioned in the API endpoint documentation https://developer.infermedica.com/docs/red-flags

Reply:
So we should update the docs, but please add that parameter, we are using it in our projects.